raritan/context.py

```diff
@@ -85,7 +86,7 @@ def set_data_reference(self, name: str, data_source) -> None:
         """
         self.data_references[name] = data_source
 
-    def get_data_reference(self, name: str):
+    def get_data_reference(self, name):
```

Comment:
why remove the :str and should we also add -> what the function returns?

Reply:
Great question! I took the type hint out of the signature because "name" changed from only ever being a string to being able to be either a string or a list. IOW you can ask for a list of data resources. If we are ok with being locked into >= Python 3.10, we could use the union type hint `string | list`. It would look like this:

```
def get_data_reference(self, name: str | list):
```

What do you think about that?

There has never been a return type because this repo doesn't want to have an opinion on the type of data you are storing. It does not have a dependency on pandas. Since we don't know what's being stored, we can't say what will be returned.

Reply:
Yes, the `str | list` is great! it matches `data: string|dict` in `test_settings` line 43. 

I see, thank you for clarifying!

Reply:
I can add those in a couple of places. I'll do that real quick.

Reply:
I addressed this in d766277473a251a4ac76010f2f3d32b57720849d